homeassistant/components/vera/__init__.py

```diff
@@ -58,6 +59,7 @@
                 vol.Required(CONF_CONTROLLER): cv.url,
                 vol.Optional(CONF_EXCLUDE, default=[]): VERA_ID_LIST_SCHEMA,
                 vol.Optional(CONF_LIGHTS, default=[]): VERA_ID_LIST_SCHEMA,
+                vol.Optional(SCENE_EXCLUDE, default=[]): VERA_ID_LIST_SCHEMA,
```

Comment:
This should not be added to the YAML configuration. We don't allow changes or additions to the YAML configuration for integrations that connect to devices or services.

Reply:
OK. I was basing this on what was there before, but happy to change it. If I need to add a new setting to the set of existing settings, what is the right way to do it?

Reply:
> If I need to add a new setting to the set of existing settings, what is the right way to do it?

I'm not sure about this question, could you clarify it?

Reply:
Sure, no problem. This component already has 3 settings in the schema - the controller URL, a list of devices to treat as lights, and a list of devices to not import. I added a 4th setting in the schema - a list of scenes to not import. I added that new setting in exactly the same way as the other settings. The change was rejected on the basis that changes or additions to the YAML configuration are now allowed. My question is if it is not allowed to add an additional setting this way, what is the correct way to do it instead? This is my first tweak to an existing component, and would appreciate pointers on how to do it right.

Reply:
You have added both the possibility to set this via YAML and via the UI (as part of an option flow). The option flow is fine 👍  The YAML part isn't allowed.

For more information see: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

Reply:
Got it - thanks for explaining! I have removed that line now and re-tested